README.md

```diff
@@ -9,78 +9,80 @@
 
 # Motivation
 The main goals are:
 
-1. To provide a high quality, well documented path to convert between data formats
-2. Support the direct use of datasets from python with minimal software
-3. Provide precise data definition via both human readable code and high quality online documentation
+1. To provide a high quality, well documented path to convert between file formats.
+2. Support the direct use of interferometric datasets from python with minimal software.
+3. Provide precise data definition via both human readable code and high quality documentation.
 
 # Package Details
 pyuvdata has four major user classes:
 
 * UVData: supports interferometric data (visibilities) and associated metadata
 * UVCal: supports interferometric calibration solutions (antenna-based) and
-associated metadata (Note that this is a fairly new object, consider it to be a beta version)
-* UVBeam: supports primary beams (E-field or power) and associated metadata
-(Note that this is a fairly new object, consider it to be a beta version)
+associated metadata.
+* UVBeam: supports primary beams (E-field or power) and associated metadata. A
+number of analytic beam objects are also available and the BeamInterface object
+serves as a unified interface for UVBeam and analytic beam objects.
 * UVFlag: A class to handle the manipulation and combination of flags for data sets.
 Also can convert raw data quality metrics into flags using thresholding.
-(This object is very new and experimental. Consider it to be a beta version)
+(This object is somewhat new and experimental. Consider it to be a beta version)
 
 ## UVData File standard notes
-* miriad has been thoroughly tested with aipy-style miriad files and minimally
-tested with ATCA files
-* uvfits conforms to AIPS memo 117 (as of March 2020).  It is tested against
-FHD, CASA, and AIPS. However AIPS is limited to <80 antennas and CASA uvfits
-import does not seem to support >255 antennas. Users with data sets containing > 255
-antennas should use the measurement set writer instead.
-* CASA measurement sets, primarily conforming to [CASA Memo 229](https://casa.nrao.edu/Memos/229.html), with some elements taken from the proposed v3.0 format documented in [CASA Memo 264](https://casacore.github.io/casacore-notes/264.html). Measurement sets are tested against
-VLA and MWA data sets, (the latter filled via cotter), with some manual verification
-haven been performed against ALMA and SMA data sets, the latter filled using the `importuvfits` task of CASA.
-tested against ALMA-filled datasets and with SMA datasets
-* uvh5 is an HDF5-based file format defined by the HERA collaboration,
+* Miriad has been thoroughly tested with aipy-style Miriad files and minimally
+tested with ATCA files. Reading/writing Miriad files is not supported on Windows.
+* UVFITS conforms to AIPS memo 117 (as of March 2020).  It is tested against
+FHD, CASA, and AIPS. However AIPS is limited to <80 antennas and CASA's ``importuvfits``
```

Comment:
I'd suggest making this warning stronger and suggesting that ``importuvfits`` be avoided in favor of `write_ms`

Reply:
done